src/configuracion.ejemplo.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/* Hacer una copia de este archivo a configuracion.php para que se use */
+
+self::$configuracion = array_merge(
+    self::$configuracion,
+    [
+        'dsn'      => 'mysql:host=localhost;dbname=asc;charset=utf8',
+        'usuario'  => 'root',
+        'clave'    => '',
+        'opciones' => [
+        ],
+    ]
+);
```

Comment:
__[PHP_CodeSniffer]__ Expected 1 blank line at end of file; 2 found ([link](https://sider.review/gh/repos/52858877/pulls/33#issue-3868029))

<sub>posted by Sider</sub>